config/config.md

```diff
@@ -319,6 +319,8 @@
 | `selector` | String | `round_robin` | Datanode selector type.<br/>- `round_robin` (default value)<br/>- `lease_based`<br/>- `load_based`<br/>For details, please see "https://docs.greptime.com/developer-guide/metasrv/selector". |
 | `use_memory_store` | Bool | `false` | Store data in memory. |
 | `enable_region_failover` | Bool | `false` | Whether to enable region failover.<br/>This feature is only available on GreptimeDB running on cluster mode and<br/>- Using Remote WAL<br/>- Using shared storage (e.g., s3). |
+| `node_expiry_tick` | String | `1minute` | Interval for checking expired nodes. |
```

Comment:
We can simply use a hardcoded value instead of exporting this option. The user doesn't need to care about it.

Reply:
> We can simply use a hardcoded value instead of exporting this option. The user doesn't need to care about it.

fixed in https://github.com/GreptimeTeam/greptimedb/pull/5592/commits/ccb40c67c5b4b1d089fc0a6ac25ff583b957f42e